-meeting-template.md

```diff
@@ -48,10 +48,10 @@ Notetaker: ...
 <!-- Links -->
 [template]: https://link.hypha.coop/template
 [meetings]: https://link.hypha.coop/meetings
-[cal]: https://calendar.google.com/calendar/embed?src=s2224p8sptnujs736vplf9anjo%40group.calendar.google.com&ctz=America%2FToronto
+[calendar]: https://link.hypha.coop/calendar
 [gh]: https://github.com/hyphacoop/organizing
-[gdrive]: https://drive.google.com/drive/u/0/folders/14KYnYwOEK3InYZ3jCn-Gtf5q430sE9oc
-[biz-wg]: https://loomio.hypha.coop/g/ojZI2bPl/working-groups-business-planning
-[fin-wg]: https://loomio.hypha.coop/g/sRPwaorg/working-groups-finance
-[gov-wg]: https://loomio.hypha.coop/g/BaAj6dQn/working-groups-governance-by-laws-incorporation-articles-gm-
-[ino-wg]: https://loomio.hypha.coop/g/KvARWad7/working-groups-infrastructure-and-operations
+[gdrive]: https://link.hypha.coop/gdrive
+[biz-wg]: https://link.hypha.coop/biz-wg
+[fin-wg]: https://link.hypha.coop/fin-wg
+[gov-wg]: https://link.hypha.coop/gov-wg
+[ops-wg]: https://link.hypha.coop/ops-wg
```

Comment:
I am pro-shortlink and prefer the shortlinks mapping is on a plaintext file in github like a yaml:

```
gdrive: https://drive.google.com/drive/u/0/folders/14KYnYwOEK3InYZ3jCn-Gtf5q430sE9oc
biz-wg: loomio.hypha.coop/g/ojZI2bPl/working-groups-business-planning
fin-wg: https://loomio.hypha.coop/g/sRPwaorg/working-groups-finance
gov-wg: https://loomio.hypha.coop/g/BaAj6dQn/working-groups-governance-by-laws-incorporation-articles-gm-
ops-wg: https://loomio.hypha.coop/g/KvARWad7/working-groups-infrastructure-and-operations
```

@dcwalk @ASoTNetworks I like to hear your thoughts on this.

Reply:
Not a hard sell on this, but the rationale was this: we learn that they exist through seeing them on link hover, not going out of the way to browse the spreadsheet :)

Will defer to collective on this though!

Reply:
I'm fine on this!